src/runtimes/docker/contracts/ContainerClient.ts

```diff
@@ -609,6 +614,30 @@ type InspectImagesCommand = {
     inspectImages(options: InspectImagesCommandOptions): Promise<PromiseCommandResponse<Array<InspectImagesItem>>>;
 };
 
+export type ImageSbomItem = {
+    /**
+     * The RAW inspect output
```

Comment:
Unless we plan on trying to parse the SBOM output, we should probably just return a string rather than an ImageSbomItem.

Reply:
yeah I agree with you here